static/sass/_patterns_takeunders.scss

```diff
@@ -0,0 +1,23 @@
+/// homepage takeunders
+@mixin maas-p-takeunders {
+  .p-takeunder {
+    border-radius: .125rem;
+    color: $color-x-light;
```

Comment:
I’ve pulled this pattern from ubuntu.com, it isn’t a strip and the background colours are set inline on the divs.